Source/MonoGame.Extended/Timers/ContinuousClock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MonoGame.Extended.Timers
+{
+    public class ContinuousClock : IGameComponent
```

Comment:
That said, I just realized that in order to have IGameComponent work in that method, I would have also needed to implement IUpdateable. Right now all it would do is set it up on the Initialize.base.

So either I can Implement IUpdatable, or I can just go ahead and remove the IGameComponent